### DISCUSSION TITLE
Blur event gets called on focus. Why can't we open an issue?

### DISCUSSION THREAD

**Discussion by cannahum** [joshnuss/svelte-stripe#58] [General]
We're using the Elements and Card components from this library (`import { Elements, Card } from 'svelte-stripe';`)
We're hooking up to focus, change and blur events. We notice that when user clicks on the input, the blur gets fired. This throws off our error tracking state. Please help.
Thanks!

**Comment by joshnuss** [comment]
Hi @cannahum,

Thanks for reporting the problem with creating issues, I've fixed that now.

As for the issue with `blur`: are you sure the `blur` event is coming from the element being clicked and not the element that is losing focus because of the click?

**Reply by cannahum** [reply]
Thanks!

We had a weird behavior where on my environment the reported issue occurs, but on someone else's it doesn't. We're looking into it to see if a Chrome Extension is interfering. I'll report here if something happens. Thanks for being responsive!

**Reply by joshnuss** [reply]
Welcome,

Try testing it in incognito mode to eliminate the possibility of extensions interfering.
